share/pot/term.sh

```diff
@@ -26,28 +26,22 @@ pot-term()
 	local _pname _force
 	_pname=
 	_force=
-	args=$(getopt hvf $*)
-	if [ $? -ne 0 ]; then
-		term-help
-		exit 1
-	fi
 
-	set -- $args
-	while true; do
-		case "$1" in
-		-h)
+	while getopts "hvf" _o; do
+		case "$_o" in
+		h)
 			term-help
-			exit 0
+			${EXIT} 0
 			;;
-		-v)
+		v)
 			_POT_VERBOSITY=$(( _POT_VERBOSITY + 1))
 			shift
 			;;
-		-f)
+		f)
 			_force="YES"
 			shift
 			;;
-		--)
+		?)
 			shift
```

Comment:
All `shift` should be removed, because `getopts` doesn't work like `getopt`
The subsequent initialization of `_pname` can be done like in `start.sh`:
```
_pname="$( eval echo \$$OPTIND)"
```

Reply:
Why eval?

Reply:
`OPTIND` contains the index, in the arguments vector, of the next argument that has to be parsed. In this case, at the end of the `case-esac` block, the pot name can be in `$1` or in `$2` or even in `$3` (if `-f` and `-v` are both selected).

`OPTIND` will contain the position where the pot name argument is stored.
`eval` allows you to do a two-step evaluation:
first step: `\$$OPTIND` => `$1` or `$2` or `$3`
second step: the content of the positional variable

currently, if you try `pot term -f potname` it will fail, because of `_pname=$1`.

Reply:
oh right